packages/grafana-ui/src/utils/dataLinks.ts

```diff
@@ -4,8 +4,11 @@ import { LinkModelSupplier } from '@grafana/data';
 export const DataLinkBuiltInVars = {
   keepTime: '__url_time_range',
   includeVars: '__all_variables',
-  seriesName: '__series_name',
-  valueTime: '__value_time',
+  seriesName: '__series.name',
+  fieldName: '__field.name',
+  valueTime: '__value.time',
+  valueNumeric: '__value.numeric',
+  seriesRefId: '__series.refId',
```

Comment:
Can't think of a use case fo refId just yet, so I think we can wait until we do.